.circleci/config.yml

```diff
@@ -69,9 +74,45 @@ workflows:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
       - golang-ci/build-and-test:
-          name: default-integration-tests
+          name: integration-tests-default
+          filters: *only_branches
+
+      - golang-ci/build-and-test:
+          name: integration-tests-with-neo4j
+          executor-name: golang-ci/default-with-neo4j
+          filters: *only_branches
+
+      - golang-ci/build-and-test:
+          name: integration-tests-with-elasticsearch
+          executor-name: golang-ci/default-with-elasticsearch
+          filters: *only_branches
+
+      - golang-ci/build-and-test:
+          name: integration-tests-with-mysql
+          executor-name: golang-ci/default-with-mysql
+          filters: *only_branches
+
+      - golang-ci/build-and-test:
+          name: integration-tests-with-postgres
+          executor-name: golang-ci/default-with-postgres
           filters: *only_branches
 
       - golang-ci/docker-build:
           name: docker-job-integration-tests
           filters: *only_branches
+
+  # Republish the dev:alpha orb every two months to ensure new pipelines don't get rejected due to expired dev orbs.
+  # We need to refer to the dev release of the orb for the integration-tests workflow. However when there is no dev orb
+  # release (or it's expired) this config file is marked as invalid and none of its workflows is allowed to run.
+  keep-dev-orb-alive:
+    triggers:
+      - schedule:
+          cron: "0 0 1 1,3,5,7,9,11 *"
```

Comment:
I'm not sure but it seems to me than one field is missing
0 - seconds
0 - minutes
1 - I guess this is the day of the month
1,3,5,7,9,11 - the month
Shouldn't we have hours before the day of the month?

Reply:
There are no seconds in cron schedule syntax - [the schedule](https://crontab.guru/#0_0_1_1,3,5,7,9,11_*) and [wiki article](https://en.wikipedia.org/wiki/Cron).